nixos/tests/acme.nix

```diff
@@ -79,15 +108,48 @@ in import ./make-test-python.nix {
             '';
           };
         })
+        ({pkgs, config, nodes, lib, ...}: {
+          security.acme.certs."example.com" = {
+            domain = "*.example.com";
+            dnsProvider = "exec";
+            dnsPropagationCheck = false;
+            credentialsFile = with pkgs; writeText "wildcard.env" ''
+              EXEC_PATH=${dnsScript { inherit writeScript bash curl; dnsAddress = nodes.dnsserver.config.networking.primaryIPAddress; }}
+            '';
+            user = config.services.nginx.user;
+            group = config.services.nginx.group;
+          };
+          systemd.targets."acme-finished-example.com" = {};
+          systemd.services."acme-example.com" = {
+            wants = [ "acme-finished-example.com.target" ];
+            before = [ "acme-finished-example.com.target" "nginx.service" ];
+            wantedBy = [ "nginx.service" ];
+          };
```

Comment:
Same here - I wouldn't expect to have to specify that.

Reply:
nginx was starting after the certs consistently when I was testing, causing the HTTP-01 validation to fail. Roll it back and run `nix-build acme.nix` and you'll see what I mean

Reply:
That's only for the wildcard certificate, right?

Reply:
No it was for the HTTP-01 challenge certs, aka the ones using `enableACME`